README.md

```diff
@@ -590,6 +598,8 @@ private unsafe void WriteEventImproved<TValue>(int eventId,
 
 I've already hinted at `Span<T>` in the previous parts. With `Span<T>` but also with the `in` paramater modifiers and `readonly struct` we can minimize the amount of copying required when operating various chunks of memory. `Span<T>` is a value type that enables the representation of contiguous regions of arbitrary memory, regardless of whether that memory is associated with a managed object, is provided by native code via interop, or is on the stack. Interally it is a pointer to a memory location and a length to represent the length of the memory represented by the span. One of the other benefits `Span<T>` provides that that because it can be "sliced" into various chunks you can represent various slices of memory of variable length without having to copy the memory. `Span<T>` can only live on the stack while its cousin `Memory<T>` can live on the heap and therefore be used in async methods for example.
 
+> I understood, but overall you lost me at this point. Is this too much? 
```

Comment:
Do you remember why I lost you with the span explanation? Was it for you not relevant? Not helpful? Too long? Too cryptic or all of that? Are you saying if I remove this explanation I wouldn't be loosing much? I was wondering how else I would introduce the next sample I'm showing.

Reply:
For my level, it's becoming too complex. And it starts to follow into the category "maybe this is too much optimization and there are areas where I can improve with less (cognitive) effort."

Based on that, I'm wondering if this part has the risk of becoming overwhelming at the end of the talk.

Reply:
I'll think about this a bit more. I thought overall it is a very good things are coming together sample but I realize the explanation there will take some time. I have removed parts of the LINQ stuff to lessen the cognitive burden. Maybe that is already good enough to ease things a bit in total. That being said, I will still consider entirely dropping it. Thanks for the feedback